.gitignore

```diff
@@ -342,8 +342,12 @@ healthchecksdb
 # Geocortex ViewerFramework
 /ViewerFramework
 
+# VertiGIS ViewerSpec
+/ViewerSpec
+
 # Geocortex.Mobile Image assets
 /**/Resources/Icons.xcassets/**/*.*
 
 # Geocortex.Icons Image assets
 /**/Resources/GcxMobileIcons.xcassets/**/*.*
+/App1/App1.Android/Resources/Resource.designer.cs
```

Comment:
Can we ignore `Resource.designer.cs` generically? This ignore pattern will stop working as soon as someone renames their project (which we expected they will want to do).